### ISSUE TITLE
Upgrade USWDS to fix mobile navbar accessibility for NVDA screen readers

### ISSUE THREAD

**Issue by raftmsohani** [raft-tech/TANF-app#2307]
## Description

***Problem***
The navbar items in mobile mode are hidden from [NVDA](https://www.nvaccess.org/download/) screen reader.

***Cause*** 
Following issue #1984, it is evident that the source of the problem is caused by a bug in [USWDS](https://designsystem.digital.gov) package version 2.13.3 currently being used in the project (see also email below). The mobile navbar sandwich button adds ```aria-hidden="true"``` to all the parent elements, and since aria-hidden hides all accessibility features from the element and all its children (see [this](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus/)), the navbar items will be hidden from screen reader.

<img width="1202" alt="Screen Shot 2022-12-07 at 7 00 58 PM" src="https://user-images.githubusercontent.com/97037188/206323421-a4d90552-833b-4980-9b7f-2ff6024ce6ea.png">

***Solution***

Upgrade USWDS to version 3.2.0.

## Deliverables

1. Upgrade USWDS to 3.2.0
2. Use forward instead of Import updated SASS from USWDS according to USWDS documentation.
3. Build frontend using Webpack following USWDS documentation. Note: The build script should be updated to include build paths for new USWDS (now ./node_modules/@uswds/uswds). 

## Acceptance Criteria
1. Tabing through navbar, all items should be readable by NVDA screen reader.
2. Test other pages with screen reader
4. Additional acceptance test for other frontend scripts and components to ensure no collateral negative affect on other items

**Notes:** 

- _Note 1_: NVDA is a MS Windows program, as such the test has to be done in Windows env

**Comment by robgendron** [comment]
@reitermb Is this related to Parity?

**Comment by reitermb** [comment]
> @reitermb Is this related to Parity?

This is a relatively high priority accessibility issue but isn't technically parity.

**Comment by lhuxraft** [comment]
May be fixed after React USWDS upgrade? Initial testing is promising
